packages/ripple-keypairs/src/types.ts

```diff
@@ -0,0 +1,4 @@
+export type ByteArray = number[] | Uint8Array
+export type HexString = string
```

Comment:
Why give this a different name?

Reply:
I think so that when you are using it in the idea it is clear that it is accepting hex.

Reply:
We should probably use some of those paradigms in xrpl.js/xrpl-py to make things more clear for people reading documentation (e.g. for memos or NFT URIs).